opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -1160,6 +1160,7 @@ namespace Opm
                 const double thp = WellBhpThpCalculator(*this).calculateThpFromBhp(well_rates_bhp_limit,
                                                                                    bhp_limit,
                                                                                    this->connections_.rho(),
+                                                                                    // TODO: why not [this]() { return well_.wellEcl().alq_value(); } like WellBhpThpCalculator.updateThp()
```

Comment:
Maybe we should also use `this->getALQ(well_state)` for the WellBhpThpCalculator::upateThp()?

@bska , do you have some idea? We should unify the code to avoid confusion if we can.

Reply:
> Maybe we should also use `this->getALQ(well_state)` for the `WellBhpThpCalculator::upateThp()`?

Yes, I think so.